core/src/main/java/binnie/core/gui/controls/button/ControlButton.java

```diff
@@ -57,11 +57,13 @@ public String getText() {
 	@Override
 	@SideOnly(Side.CLIENT)
 	public void onRenderBackground(int guiWidth, int guiHeight) {
-		Object texture = CraftGUITexture.BUTTON_DISABLED;
+		Object texture;
```

Comment:
One nice use of `final` is for cases like this. You can make `final Object texture;` and the static analyzer will complain "variable 'texture' might not have been initialized" if there is a case where it is not set.

Reply:
I introduced the proposed changes, but it seems that the static analyzer worked correctly without them.